crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -60,14 +634,16 @@ impl From<[u8; ADDRESS_LEN]> for Address {
 
 impl Display for Address {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        use hex::ToHex as _;
-        f.write_str(&self.encode_hex::<String>())
+        for byte in self.0 {
+            write!(f, "{byte:02x}")?;
+        }
```

Comment:
is this better than using `encode_hex`?

Reply:
@noot forgot to answer: this avoids an allocation. The hex crate only has methods that allocate to a string, whereas this writes directly to the formatter without an extra allocation.